src/app/plugins/listclasses.plugin.ts

```diff
@@ -41,7 +42,7 @@ export class ListClassesPlugin extends Plugin {
       for (const r of response) {
         try {
           await message.reply(r);
-        } catch (e) {}
+        } catch (e) { }
```

Comment:
Are you using a different format config ??

Reply:
I'm only using the default I'll change it for future.